### ISSUE TITLE
[CY-5814] Codacy RuboCop breaks when loading cops from `graphql` gem

### ISSUE THREAD

**Issue by dimerman** [codacy/codacy-rubocop#300]
my rubocop.yml
```
require:
  - rubocop-graphql
  - rubocop-performance
  - rubocop-rails
  - rubocop-rspec
  - rubocop-sorbet
  - ./lib/rubocop/cop/recital
  - graphql/rubocop/graphql/default_null_true
  - graphql/rubocop/graphql/default_required_true
```

codacy error:
```
KubernetesDockerRunner: Container for codacy/codacy-rubocop:5.4.52 exited with non-zero code 1
Error executing the tool
java.lang.Exception: 
Rubocop exited with code 2
stdout: 
stderr: cannot load such file -- graphql/rubocop/graphql/default_null_true
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader_resolver.rb:17:in `block (2 levels) in resolve_requires'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader_resolver.rb:13:in `each'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader_resolver.rb:13:in `block in resolve_requires'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader_resolver.rb:12:in `tap'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader_resolver.rb:12:in `resolve_requires'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader.rb:44:in `load_file'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_loader.rb:103:in `configuration_from_file'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_store.rb:68:in `for_dir'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/config_store.rb:47:in `for_pwd'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:146:in `all_cops_include'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:129:in `ruby_extensions'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:124:in `ruby_extension?'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:169:in `ruby_file?'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:177:in `included_file?'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:184:in `block in process_explicit_path'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:184:in `select!'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:184:in `process_explicit_path'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:40:in `block in find'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:36:in `each'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/target_finder.rb:36:in `find'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/runner.rb:77:in `find_target_files'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/runner.rb:42:in `run'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/command.rb:11:in `run'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli/environment.rb:18:in `run'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli.rb:71:in `run_command'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli.rb:78:in `execute_runners'
/usr/local/bundle/gems/rubocop-1.23.0/lib/rubocop/cli.rb:47:in `run'
/usr/local/bundle/gems/rubocop-1.23.0/exe/rubocop:12:in `block in <top (required)>'
/usr/lib/ruby/2.7.0/benchmark.rb:308:in `realtime'
/usr/local/bundle/gems/rubocop-1.23.0/exe/rubocop:12:in `<top (required)>'
/usr/local/bundle/bin/rubocop:23:in `load'
/usr/local/bundle/bin/rubocop:23:in `<main>'
                
	at codacy.rubocop.Rubocop$.apply(Rubocop.scala:57)
	at com.codacy.tools.scala.seed.DockerEngine.executeTool(DockerEngine.scala:53)
	at com.codacy.tools.scala.seed.DockerEngine.$anonfun$main$2(DockerEngine.scala:35)
	at scala.util.Success.map(Try.scala:262)
	at com.codacy.tools.scala.seed.DockerEngine.$anonfun$main$1(DockerEngine.scala:27)
	at scala.util.Success.flatMap(Try.scala:258)
	at com.codacy.tools.scala.seed.DockerEngine.main(DockerEngine.scala:26)
	at codacy.Engine.main(Engine.scala)
```

**Comment by machadoit** [comment]
Hi @dimerman can you help me to find that cop?

From my understanding on the README of the graphql plugin: "All cops are located under [lib/rubocop/cop/graphql](https://github.com/DmitryTsepelev/rubocop-graphql/blob/master/lib/rubocop/cop/graphql), and contain examples and documentation". And on that folder, I don't see any `default_null_true`.

Also, the suggestion is to add the cop on the .rubocp.yml and they provide an example like `GraphQL/ResolverMethodLength`, so can you try that instead of the path to the file?

Are you able to run Rubocop by itself with no problems, when it is installed on your machine? 

One last thing, can you try to run the docker itself on top of your repository? If you have docker installed, from the root of your repository, can you check the results of:

```
docker run -it -v $(pwd):/src  codacy/codacy-rubocop:5.4.57
```

**Comment by dimerman** [comment]
please have a look at https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/rubocop/graphql/default_null_true.rb#L23

**Comment by machadoit** [comment]
Regarding the other questions, did you have the opportunity to check? Like adding it to the config file as a `DefaultNullTrue` with the module of the Cop instead of the location of the file?

Probably we may need to add `graphql` itself as part of our https://github.com/codacy/codacy-rubocop/blob/master/Gemfile 

Do you have any open source repo where we can try to fix it to be able to validate? Also, feel free to check our instructions at https://github.com/codacy/codacy-rubocop#readme and contribute with a PR if you know the solution.

**Comment by dimerman** [comment]
I agree: adding `graphql` itself to the gemfile sounds like the way to go.

I'm afraid I don't know of any open source repository where you can try the fix. 
However I think any new (empty) rails project with this gems and config will do for validation purposes.

**Comment by machadoit** [comment]
Fixed by #305, so the tool no longer crashes when referencing graphql files on the configuration
